freqtrade/rpc/telegram.py

```diff
@@ -205,13 +206,14 @@ def _format_sell_msg(self, msg: Dict[str, Any]) -> str:
         msg['emoji'] = self._get_sell_emoji(msg)
 
         message = ("{emoji} *{exchange}:* Selling {pair} (#{trade_id})\n"
+                   "*Profit:* `{profit_percent:.2f}%`\n"
+                   "*Sell Reason:* `{sell_reason}`\n"
+                   "*Duration:* `{duration} ({duration_min:.1f} min)`\n"
                    "*Amount:* `{amount:.8f}`\n"
                    "*Open Rate:* `{open_rate:.8f}`\n"
                    "*Current Rate:* `{current_rate:.8f}`\n"
-                   "*Close Rate:* `{limit:.8f}`\n"
-                   "*Sell Reason:* `{sell_reason}`\n"
-                   "*Duration:* `{duration} ({duration_min:.1f} min)`\n"
-                   "*Profit:* `{profit_percent:.2f}%`").format(**msg)
+                   "*Close Rate:* `{limit:.8f}`"
+                   ).format(**msg)
```

Comment:
I would think that you've changed this in the wrong location.

This method is for sell notifications - not for `/profit` answers.
This change (to me, not related to this PR) is what breaks CI now ...

Reply:
Oh, sorry, I accidentally committed my experiments
Will revert them asap